portal-impl/src/com/liferay/portal/verify/VerifyProperties.java

```diff
@@ -309,6 +309,10 @@ protected static void verifySystemProperties() throws Exception {
 			"http.header.secure.x.frame.options.255",
 			"http.header.secure.x.frame.options.255"
 		},
+		{
+			"liferay.shielded.container.lib.portal.dir",
```

Comment:
@W2jx There's no such property in our portal properties file, this is redundant. This property is completely set at runtime, when starting the JSP engine, so there's nothing to migrate in properties file.